lib/kvdag/edge.rb

```diff
@@ -1,8 +1,20 @@
 class KVDAG
+  
+  # An edge to a vertex in a +KVDAG+
+
   class Edge
     include AttributeNode
+
+    # Return the target vertex of this edge
+
     attr_reader :to_vertex
 
+    # Create a new edge towards a vertex in a +KVDAG+, 
+    # optionally loaded with key-values
+    #
+    # N.B: <tt>KVDAG::Edge.new</tt> should never be called directly,
+    # always use <tt>KVDAG::Vertex#edge</tt> to create edges.
+
```

Comment:
If you just write the symbol names, without `+` or `<tt>`, rdoc will generate links to their documentation.